shared/settings/files/index.js

```diff
@@ -1,28 +1,110 @@
+  100 * 1024 ** 2,
+  1024 ** 3,
+  10 * 1024 ** 3,
+]
+
+export const defaultNotificationThreshold = 100 * 1024 ** 2
+
 const EnableSystemFileManagerIntegration = (props: Props) => (
   <Kb.Box style={Styles.globalStyles.flexBoxColumn}>
-    <Kb.Text type="Body">Enable Keybase in {fileUIName}</Kb.Text>
+    <Kb.Text type="Body">Enable Keybase in {Platform.fileUIName}</Kb.Text>
     <Kb.Text type="BodySmall">
       Access your Keybase files just like you normally do with your local files.
     </Kb.Text>
   </Kb.Box>
 )
 
+class ThresholdDropdownMobile extends React.PureComponent<Props, {visible: boolean}> {
+  state = {visible: false}
+  _show = () => this.setState({visible: true})
+  _hide = () => this.setState({visible: false})
+  _select = selectedIdx => this.props.onChangedSyncNotifications(selectedIdx)
+  render() {
+    return (
+      <>
+        <InlineDropdown
+          onPress={this.state.visible ? this._show : this._hide}
+          type="Body"
+          label={Constants.humanizeBytes(this.props.spaceAvailableNotificationThreshold || defaultNotificationThreshold, 0)}
+        />
+        <Kb.FloatingPicker
+          items={allowedNotificationThresholds.map(i => ({
+            label: Constants.humanizeBytes(i, 0),
+            value: i,
+          }))}
+          visible={this.state.visible}
+          selectedValue={
+            this.props.spaceAvailableNotificationThreshold || defaultNotificationThreshold
+          }
+          promptString="Pick a threshold"
+          prompt={
+            <Kb.Box2 direction="horizontal" fullWidth={true} gap="xtiny" centerChildren={true}>
+              <Kb.Text type="BodySmallSemibold">Pick a threshold</Kb.Text>
+            </Kb.Box2>
+          }
+          onCancel={this._hide}
+          onHidden={this._hide}
+          onDone={this._hide}
+          onSelect={this._select}
+        />
+      </>
+    )
+  }
+}
+
+const ThresholdDropdownDesktop = (props: Props) => (
+  <Kb.Dropdown
+    items={allowedNotificationThresholds.map(i => (
+      <Kb.Text type="Body" key={i} >{Constants.humanizeBytes(i, 0)}</Kb.Text>
+    ))}
+    onChangedIdx={props.onChangedSyncNotifications}
+    selected={(
+      <Kb.Box2 direction="horizontal" key={props.spaceAvailableNotificationThreshold || defaultNotificationThreshold}>
+        <Kb.Text type="Body">{Constants.humanizeBytes(props.spaceAvailableNotificationThreshold || defaultNotificationThreshold, 0)}</Kb.Text>
+      </Kb.Box2>
+    )}
+    style={styles.syncNotificationSettingDropdown}
+    selectedBoxStyle={styles.syncNotificationDropdownItem}
+    disabled={props.areSettingsLoading || props.spaceAvailableNotificationThreshold === 0}
+  />
+)
+
+const ThresholdDropdown = Platform.isMobile ? ThresholdDropdownMobile : ThresholdDropdownDesktop
+
+const SyncNotificationSetting = (props: Props) => (
+  <Kb.Box2 direction="horizontal" alignItems="center">
+    <Kb.Text type="Body">Warn me if I only have less than </Kb.Text>
```

Comment:
"only have less than" feels redundant to me. Maybe cut the 'only'?

Reply:
Copy is from design. I'll change it though to "Warn me if I have less than <n> storage space remaining".